### ISSUE TITLE
Autocomplete in Demo returns unavailable functions

### ISSUE THREAD

**Issue by dvd101x** [josdejong/mathjs#3004]
Hi, I've been checking the autocomplete in the Demo and found that for example `isUn` autocompletes to `isUnit` and pressing enter it returns `Error: Undefined symbol isUnit` 

There are more than 80 functions in mathjs that are not available in the parser. The autocomplete does include them even though they will throw an error.

To replicate try starting to type any of the following and pressing tab to autocomplete and enter

```js
[
    "isNumber", "isComplex", "isBigNumber", "isFraction", "isUnit", "isString", "isArray", "isMatrix",
    "isCollection", "isDenseMatrix", "isSparseMatrix", "isRange", "isIndex", "isBoolean", "isResultSet",
    "isHelp", "isFunction", "isDate", "isRegExp", "isObject", "isNull", "isUndefined", "isAccessorNode",
    "isArrayNode", "isAssignmentNode", "isBlockNode", "isConditionalNode", "isConstantNode",
    "isFunctionAssignmentNode", "isFunctionNode", "isIndexNode", "isNode", "isObjectNode",
    "isOperatorNode", "isParenthesisNode", "isRangeNode", "isRelationalNode", "isSymbolNode",
    "isChain", "on", "off", "once", "emit", "import", "create", "factory", "AccessorNode", "apply",
    "ArrayNode", "AssignmentNode", "BigNumber", "BlockNode", "Chain", "Complex", "ConditionalNode",
    "ConstantNode", "DenseMatrix", "FibonacciHeap", "Fraction", "FunctionAssignmentNode", "FunctionNode",
    "Help", "ImmutableDenseMatrix", "Index", "IndexNode", "Matrix", "Node", "not", "ObjectNode",
    "OperatorNode", "ParenthesisNode", "Parser", "Range", "RangeNode", "RelationalNode", "ResultSet", 
    "Spa", "SparseMatrix", "SymbolNode", "Unit", "ArgumentsError", "DimensionError", "IndexError"
]
```

The code to get the previous list was:

```js
Object.keys(math)
    .filter(x => typeof math[x] === "function")
    .filter(x => {
        try {
            math.evaluate(x)
            return false
        }
        catch {
            return true
        }
    })
```

**Comment by dvd101x** [comment]
I think the `cli.js` method would fix the one in the Demo

https://github.com/josdejong/mathjs/blob/49c793ba5a47359157a2c5009e63b11e43fcff2e/bin/cli.js#L114-L120

**Comment by josdejong** [comment]
Thanks for reporting David. Fixed now on the website via 67a9784.
